data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,279 @@ event "remnant: pantica void sprites"
+			choice
+				`	"Good work. What kind of data are you recording?`
+				`	"Sounds good. Just let me know when you are ready to leave."`
+					goto astraldone
+			`	She smiles, and replies. "It might be easier to ask what we are leaving out. The data we are collecting broadly falls into three sets: data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor can we take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`
+			choice
+				`	"So this is not the first expedition here?"`
+				`	"Anything unusual in the data?"`
+					goto astraldata
+				`	"How much longer does the equipment need?"`
+					goto astralfinish
+			`	"No, this is the forty-third such expedition. Sagittarius A* was one of the first places our explorers sought to visit after we acquired jump drives, and we have been coming back regularly ever since. I know some of our reconnaissance pilots consider visiting here a pilgrimage of sorts. It is humbling to be here myself after reviewing so many data sets of past expeditions." She looks thoughtful, staring out the window at the the hub of the galaxy, as if her thoughts are jumping from idea to idea.`
+			choice
+				`	"Anything unusual in the data?"`
+				`	"How much longer does the equipment need?"`
+					goto astralfinish
+			label astraldata
+			`	"There are many oddities about this place. Every visit helps revise our physics models a little closer. But they also reveal problems." She frowns, her gestures taking on a curious tone. "For instance, you will note that your ramscoops are not picking up anything. According to standard astronomical theory, this system should be a veritable river of suitable material, and yet there is nothing. Not even the normal background trace amounts we find in deep space. We have visited other black holes, and they conform to the standard models in this respect. But not Sagittarius A*." She pauses, thinking about it for a moment.`
+			choice
+				`	"Is the equipment finished?"`
+				`	"How much longer does the equipment need?"`
+			label astralfinish 
+			`	"The equipment will be finished shortly, Captain <first>. I just need to make a few observations, and then we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`
+			`	"I will be going outside for a few minutes," Dawn declares. "I shall just be on the hull."`
+			choice
+				`	"Let me know when you are done."`
```

Comment:
```suggestion
				`	"Let me know when you're done."`
```

Reply:
Same as the previous contraction. Long story short, I am deliberately avoiding using contractions for both sides of Remnant dialog.

I realize this does make it sound slightly odd, because we are used to having those contractions there. This oddity is intentional, and is a subtle hint that their thought patterns are at least a _little_ different.